src/scope.rs

```diff
@@ -99,8 +99,8 @@ macro_rules! impl_scope_ext {
             /// Ignores passed `wgpu::RenderPassDescriptor::timestamp_writes` and replaces it with
             /// `timestamp_writes` managed by `GpuProfiler`.
             ///
-            /// Note that in order to take measurements, this does not require the
-            /// [`wgpu::Features::TIMESTAMP_QUERY_INSIDE_PASSES`] feature, only [`wgpu::Features::TIMESTAMP_QUERY`].
+            /// Note that in order to take measurements, this requires the
+            /// [`wgpu::Features::TIMESTAMP_QUERY_INSIDE_ENCODERS`] feature.
```

Comment:
this is wrong. Scoped renderpasses need `wgpu::Features::TIMESTAMP_QUERY` and not the other two additional features. (This is the only thing that works on WebGPU)

Reply:
the subsequent checking is incorrect as well.
Feels like this also requires extending unit testing on this a bit